client/stylesheets/_toCat.scss

```diff
@@ -2760,11 +2760,17 @@ i:hover {
     display: inline-block !important;
 }
 
+.managerReceivedOffers {
+	display: inline-block;
+}
+
 .managerOfferCount {
     padding: 10px 18px !important;
     background-color: #00bfff !important;
+    color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces